tools/install.py

```diff
@@ -127,7 +129,14 @@ def subdir_files(path, dest, action):
   for subdir, files in ret.items():
     action(files, subdir + '/')
 
-def files(action):
+
+def files(components, action):
+  funcs = {'headers': header_files, 'node': node_files, 'npm': npm_files}
+  for component in components:
+    funcs[component](action)
+
+
```

Comment:
Oops sorry. Actually Python style guide recommends two line gaps between functions. As the indentation is important in Python, the two line gaps make it easier to read. I'll change it, so that it will be consistent throughout the file.